Dockerfile

```diff
@@ -5,3 +5,5 @@ RUN conda install --yes -c conda-forge --file /tmp/conda-requirements.txt
 
 COPY requirements.txt /tmp/
 RUN pip install -r /tmp/requirements.txt
+
+RUN pip install -U jupyterlab
```

Comment:
Probably worth while to specify a version, and run `juptyer lab build` on the next line.